File: Modul-4/Soal-3/PRAK403-2410817110016-MuhammadAzriRahman.py
angka1, angka2 = map(int, input("Masukkan 2 angka: ").split())


a = angka1
b = angka2


if angka1 > angka2:
    for i in range(b, angka1 + 1):
        print(f"{a} {b}", end="")
        if angka1 > b:
             print(" - ", end="")
        b += 1  
        a -= 1 
    print()  


elif angka2 > angka1:
    for i in range(b, angka1 - 1, -1):
        print(f"{a} {b}", end="")
        if angka2 > a:  
            print(" - ", end="")
        b -= 1 
        a += 1 
    print()  

else:
    print(f"{a} {b}")

File: Modul-4/Soal-3/PRAK403-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int angka1, angka2;

    printf("Masukan 2 angka: ");
    scanf(" %d %d", &angka1, &angka2);

    int a = angka1, b = angka2;
    if (angka1 > angka2) {
        for (int i = b; i <= angka1; i++) {
            printf("%d %d", a, b);
            if (angka1 > b)
                printf(" - ");
            b++;
            a--;
        }
    }
    else if (angka2 > angka1) {
        for (int i = b; b >= angka1; i--) {
            printf("%d %d",a, b);
            if (angka2 > a)
                printf(" - ");
            b--;
            a++;
        }}
    else {
        printf("%d %d", a,b);
    }
}